tools/json-to-module/redirect.json

```diff
@@ -0,0 +1,99 @@
+{
+    "aatm": "https://www.dndbeyond.com/sources/dnd/aatm/the-mortuary",
+    "ai": "https://www.dndbeyond.com/sources/dnd/ai",
+    "bgdia": "https://www.dndbeyond.com/sources/dnd/bgdia",
+    "bgg": "https://www.dndbeyond.com/sources/dnd/bgg/baldurs-gate-gazetteer",
+    "boet": "https://www.dndbeyond.com/sources/dnd/boet",
+    "br": "https://www.dndbeyond.com/sources/dnd/basic-rules-2014",
+    "cm": "https://www.dndbeyond.com/sources/dnd/cm",
+    "cos": "https://www.dndbeyond.com/sources/dnd/cos",
+    "cosco": "https://www.dndbeyond.com/sources",
```

Comment:
There is 3 cases of source that we can't find: `cosco`, `ee` and `sd`. For each of these, I point at the DDB listing page: "https://www.dndbeyond.com/sources".